src/utils/log.ts

```diff
@@ -106,9 +109,9 @@ export class StandardLogger implements Logger {
     this.printer(message);
   };
 
-  public setLogConfigLevel = (logConfigLevel: LogConfigLevel): void => {
+  setLogConfigLevel(logConfigLevel: LogConfigLevel): void {
     this.logConfigLevel = logConfigLevel;
-  };
+  }
```

Comment:
Feel free to remove `public` and convert the arrow function to a classical function 👍 
However, please unify the style in the class.